cfgov/unprocessed/apps/filing-instruction-guide/js/fig-toc-frame.js

```diff
@@ -0,0 +1,32 @@
+export function buildMobileTOC() {
+    expandableTOCFrame = `<div class="o-expandable
+    o-expandable__padded
+    o-expandable__background
+    o-expandable__border">
+<button class="o-expandable_header o-expandable_target"
+    title="Expand Table Of Contents">
+<h3 class="h4 o-expandable_header-left o-expandable_label">
+    Table of Contents
+</h3>
+<span class="o-expandable_header-right o-expandable_link">
+    <span class="o-expandable_cue o-expandable_cue-open">
+        <span class="u-visually-hidden-on-mobile">Show</span>
+        {% include icons/plus-round.svg %}
```

Comment:
These will need to be the full `<svg ... >` elements because jinja won't be processing the JS files. You can get the svg code for specific icons by locating them in the DS repo and viewing the page source in Chrome. E.g. here's the [plus-round](https://raw.githubusercontent.com/cfpb/design-system/main/packages/cfpb-icons/src/icons/plus-round.svg) icon.